### PR TITLE
commit 3: fix

### PR DESCRIPTION
**Pull request by Itandr**
I think it is better.

### CODE REVIEW

**Comment by Itandr** [comment]
What do you think

**Comment by avdolgushev** [comment]
Let me think

File: 1 [hunk 1]
```diff
@@ -1 +1,19 @@
...
\ No newline at end of file
+1111
+
+
+
+
+
+
+
+
```
Comment (avdolgushev) [inline 1]: Do you think it really matters

Comment (Itandr) [reply 1]: > Do you think it really matters

yes

Comment (Itandr) [reply 1]: truly yes

Comment (Itandr) [reply 1]: ?

File: 1 [hunk 2]
```diff
@@ -1 +1,19 @@
...
+
+
```
Comment (avdolgushev) [inline 1]: Do you think it really matters

Comment (avdolgushev) [reply 1]: gmm

**Pull request merged**